shardingsphere-mode/shardingsphere-mode-type/shardingsphere-standalone-mode/shardingsphere-standalone-mode-repository/shardingsphere-standalone-mode-repository-provider/shardingsphere-standalone-mode-repository-h2/src/test/java/org/apache/shardingsphere/mode/repository/standalone/h2/H2RepositoryTest.java

```diff
@@ -34,38 +34,40 @@
     @Test
     public void assertMethod() {
         assertThat(h2Repository.getType(), is("H2"));
-        assertSetProperty();
-        assertPersistAndGet();
-        assertPersistAndGetChildrenKeys();
-        assertDelete();
         h2Repository.close();
     }
-    
-    private void assertSetProperty() {
+
+    public void assertSetProperty() {
```

Comment:
there is no `@Test` annotation, why make this public

Reply:
Hey it was a mistake I have added a new PR.
Please check.